src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { PageGhangeCallback } from '../../types/PageChangeCallback';
```

Comment:
There is a typo in the import statement for 'PageChangeCallback'. It is written as 'PageGhangeCallback' which is incorrect. Please correct the spelling to 'PageChangeCallback' to ensure proper import of the type.